Kernel/Arch/x86_64/Processor.h

```diff
@@ -270,7 +270,7 @@ ALWAYS_INLINE bool ProcessorBase<T>::has_pat() const
 }
 
 template<typename T>
-ALWAYS_INLINE u64 ProcessorBase<T>::read_cpu_counter()
+ALWAYS_INLINE u64 ProcessorBase<T>::read_cycle_count()
 {
     return read_tsc();
```

Comment:
Eh, it's a bit of a mess on x86. For older processors it's actual cycles which vary with boost level. On newer cpus it's a frequency-invariant timestamp. But I suppose the new name isn't worse.

Reply:
Oh yeah I know, I thought about adding a comment but thought it isn't really necessary.
The RISC-V cycle CSR also may or may not tick at a constant rate.